### ISSUE TITLE
aws credential issue introduced in 0.0.5

### ISSUE THREAD

**Issue by bpesics** [takus/fluent-plugin-ec2-metadata#5]
with 0.0.4 the following worked fine:
```
  <match fluentd.all>
    type ec2_metadata
    output_tag ${tag}.ec2_metadata
    <record>
      fd_meta.ec2_az ${availability_zone}
      fd_meta.ec2_instance_id ${instance_id}
    </record>
  </match>
```

with 0.0.5:
```
2015-04-27 12:08:45 +0000 [error]: unexpected error error="\nMissing Credentials.\n\nUnable to find AWS credentials.  You can configure your AWS credentials\na few different ways:\n\n* Call AWS.config with :access_key_id and :secret_access_key\n\n* Export AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY to ENV\n\n* On EC2 you can run instances with an IAM instance profile and credentials\n  will be auto loaded from the instance metadata service on those\n  instances.\n\n* Call AWS.config with :credential_provider.  A credential provider should\n  either include AWS::Core::CredentialProviders::Provider or respond to\n  the same public methods.\n\n= Ruby on Rails\n\nIn a Ruby on Rails application you may also specify your credentials in\nthe following ways:\n\n* Via a config initializer script using any of the methods mentioned above\n  (e.g. RAILS_ROOT/config/initializers/aws-sdk.rb).\n\n* Via a yaml configuration file located at RAILS_ROOT/config/aws.yml.\n  This file should be formated like the default RAILS_ROOT/config/database.yml\n  file.\n\n"
```

the two used parameters do not require API access according to the documentation

**Comment by takus** [comment]
Sorry for bothering you. This will be fixed by #6.

**Comment by takus** [comment]
v0.0.6 is released.
